model/CategoryLimitRequest.ts

```diff
@@ -0,0 +1,10 @@
+import { CategoryFundingAccountAccessRequest } from "./CategoryFundingAccountAccessRequest"
+import { UserLimitsRequest } from "./UserLimitsRequest"
+
+export interface CategoryLimitRequest {
+	limits?: UserLimitsRequest[]
```

Comment:
ive changed the object to accept setLimits, use that instead? (the old one still works too)